x-pack/plugins/cases/common/index.ts

```diff
@@ -40,13 +40,14 @@ export {
   LENS_ATTACHMENT_TYPE,
 } from './constants';
 
+export { ConnectorTypes } from './types/domain';
```

Comment:
Does this need to be here? all usages(at least in this PR) import either from `.../common/types/domain` or on the ouside from `'@kbn/cases-plugin/common/types/domain'`

Reply:
This is needed by other plugins like the Security Solution. This is why we need to export it from the `common` folder. Plugins should only import from the `common` and not go deeper in our paths.

Reply:
> This is needed by other plugins like the Security Solution. 

Exactly but then [in these changed to the security solution plugin](https://github.com/elastic/kibana/pull/162059/files#diff-755cf73864933c9810fc1b0b5a5ce13174493a6610314be43a8cdf0fc6f0b837) we import from `'@kbn/cases-plugin/common/types/domain'`.

So that should maybe be changed to import from '@kbn/cases-plugin/common'?

Reply:
Yes, you are right. I will change it.